pxemgr/ipxe_handlers.go

```diff
@@ -617,19 +615,23 @@ func (mgr *pxeManagerT) welcomeHandler(w http.ResponseWriter, r *http.Request) {
 }
 
 func (mgr *pxeManagerT) hostCoreOSVersion(r *http.Request) string {
-	coreOSversion := defaultCoreOSVersion
+	coreOSversion := mgr.config.DefaultCoreOSVersion
+	fmt.Println("default: ", coreOSversion)
 
 	host, exists := mgr.hostFromSerialVar(r)
 	if exists {
+		fmt.Println("host from serial: ", host.CoreOSVersion)
+
+		if version, exist := host.Overrides["CoreOSVersion"]; exist {
+			fmt.Println("send host override: ", version.(string))
```

Comment:
hehe, sry these are anyway just debug messages. but yes, I'll remove them and add proper logging.